src/main/java/backend/Logic.java

```diff
@@ -127,9 +127,9 @@ public void refresh() {
         });
     }
 
-    public CompletableFuture<Boolean> getIssueMetadata(String repoId, List<Integer> issues) {
+    public CompletableFuture<Boolean> getIssueMetadata(String repoId, Map<Integer, String> issues) {
```

Comment:
It's more that it's not as clear as it could be, to someone just looking at the method signature in isolation.

The metadata that the issue points to can change, but the metadata object itself is immutable, so it wouldn't change in a shallow copy. I'm more concerned about the `labels` field, which is mutated (via `addLabel`) and not always copied.

The best approach seems to be to fix the mutability issues and reuse TurboIssues, rather than make another class. What do you think?